creator-node/src/RehydrateIpfsQueue.js

```diff
@@ -0,0 +1,98 @@
+const Bull = require('bull')
```

Comment:
https://github.com/OptimalBits/bull/blob/develop/REFERENCE.md#queueprocess

Why do we want default concurrency = 1? Seems like we can handle more than 1 of these at a time.

Reply:
I wasn't sure on how many concurrent tasks to be processed. how do you determine the best count?

Reply:
discussed offline - make a config value, 10-20 is good as a starting point